app/views/static/cookies.md

```diff
@@ -0,0 +1,38 @@
+<% content_for :page_title, 'Cookies' %>
+
+# Cookies
+
+Cookies are small files saved on your phone, tablet or computer when you visit
+a website.
+
+We use cookies to make the <%= t('service.name') %> service work and
+collect information about how you use our service.
+
+## Essential cookies
+
+Essential cookies keep your information secure while you use the <%=
+t('service.name') %> service. We do not need to ask permission to use them.
+
+<table class="govuk-table">
+  <caption class="govuk-visually-hidden">Essential cookies</caption>
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th class="govuk-table__header">Name</th>
+      <th class="govuk-table__header">Purpose</th>
+      <th class="govuk-table__header">Expires</th>
+    </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell">
+        _session_id
+      </td>
+      <td class="govuk-table__cell" width="50%">
+        Used to keep you signed in
+      </td>
+      <td class="govuk-table__cell">
+        2 hours
```

Comment:
Not true just yet!

Can you open a follow-up PR to steal this file from Find? https://github.com/DFE-Digital/find-a-lost-trn/blob/main/config/initializers/session_store.rb